src/app/modules/entry-live/views/live-discovery-chart/discovery-chart/discovery-chart.component.ts

```diff
@@ -141,8 +141,22 @@ export class DiscoveryChartComponent implements OnDestroy{
     mainMin = mainMin === mainMax ? 0 : mainMin;
     secondaryMin = secondaryMin === secondaryMax ? 0 : secondaryMin;
 
-    const mainInterval = parseFloat(((mainMax - mainMin) / 5).toFixed(2));
-    const secondaryInterval = parseFloat(((secondaryMax - secondaryMin) / 5).toFixed(2));
+    let mainInterval = parseFloat(((mainMax - mainMin) / 5).toFixed(2));
+    let secondaryInterval = parseFloat(((secondaryMax - secondaryMin) / 5).toFixed(2));
+
+    // make the same scale if both have the same units
+    if (this.fields[mainMetric].units && this.fields[secondaryMetric].units &&
```

Comment:
@viktor-kyryltsev-kaltura - this could be problematic when there is a large difference between metrics values for example millions vs. small units. Lets discuss if this is required.